plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java

```diff
@@ -230,7 +230,7 @@ public DistributedQueryRunner build()
                 }
 
                 HiveMetastore metastore = this.metastore.apply(queryRunner);
-                queryRunner.installPlugin(new TestingHivePlugin(metastore, module, cachingDirectoryLister));
+                queryRunner.installPlugin(new TestingHivePlugin(Optional.of(metastore), module, cachingDirectoryLister));
```

Comment:
We should also restrict invoking `populateData` if the metastore is null.. we might need to configure it as read-only query runner.

Reply:
> We should also restrict invoking `populateData` if the metastore is null..

@Praveen2112 why?

anyway, it's always present here anyway